packages/api/test/e2e/promise-queries.spec.ts

```diff
@@ -200,43 +200,42 @@ describe.skip('Promise e2e queries', () => {
   });
 
   describe('with plain type', () => {
-    const EXISTENTIAL_DEPOSIT = 500;
-    it('queries correct value', async () => {
-      const existentialDeposit = await api.query.balances.existentialDeposit() as Balance;
+    it('queries a correct value', async () => {
+      const sessionIndex = await api.query.session.currentIndex() as SessionIndex;
 
-      expect(existentialDeposit.toNumber()).toEqual(EXISTENTIAL_DEPOSIT);
+      expect(sessionIndex.toNumber()).toBeGreaterThanOrEqual(0);
     });
 
     it('queries correct value at a specified block', async () => {
       const header = await api.rpc.chain.getHeader() as Header;
-      const existentialDepositAt = await api.query.balances.existentialDeposit.at(header.hash) as Balance;
+      const sessionIndex = await api.query.session.currentIndex.at(header.hash) as SessionIndex;
 
-      expect(existentialDepositAt.toNumber()).toEqual(EXISTENTIAL_DEPOSIT);
+      expect(sessionIndex.toNumber()).toBeGreaterThanOrEqual(0);
     });
 
     it('subscribes to query and get correct result', (done) => {
-      return api.query.balances.existentialDeposit((existentialDeposit: Balance) => {
-        expect(existentialDeposit.toNumber()).toEqual(EXISTENTIAL_DEPOSIT);
+      return api.query.session.currentIndex((sessionIndex: SessionIndex) => {
+        expect(sessionIndex.toNumber()).toBeGreaterThanOrEqual(0);
         done();
       });
     });
 
     it('queries correct hash', async () => {
-      const hash = await api.query.balances.existentialDeposit.hash();
+      const hash: Hash = await api.query.session.currentIndex.hash();
 
       expect(hash).toBeDefined();
     });
 
     it('gets correct key', async () => {
-      const key = api.query.balances.existentialDeposit.key();
-      const existentialDepositData = await api.rpc.state.getStorage(key) as Option<any>;
-      const existentialDepositRPC = new Balance(existentialDepositData.unwrapOr(undefined));
+      const key = api.query.session.currentIndex.key();
+      const sessionIndexData = await api.rpc.state.getStorage(key) as Option<any>;
+      const sessionIndexRPC = new SessionIndex(sessionIndexData.unwrapOr(undefined));
 
-      expect(existentialDepositRPC.toNumber()).toEqual(EXISTENTIAL_DEPOSIT);
+      expect(sessionIndexRPC.toNumber()).toBeGreaterThanOrEqual(0);
     });
 
     it('queries correct size', async () => {
-      const size = await api.query.balances.existentialDeposit.size();
+      const size = await api.query.session.currentIndex.size();
```

Comment:
hmm this test fails for me

```
Promise e2e queries › with plain type › queries correct size

    expect(received).not.toEqual(expected) // deep equality

    Expected: not 0

      238 |       const size = await api.query.session.currentIndex.size();
      239 | 
    > 240 |       expect(size.toNumber()).not.toEqual(0);
          |                                   ^
      241 |     });
      242 |   });
      243 | 

      at Object.toEqual (packages/api/test/e2e/promise-queries.spec.ts:240:35)
```

Reply:
which chain? And at which block?

I tested with 1.0 and substrate master and they were passing. I'll try with a fresh node.

Reply:
latest master from 3h ago, fresh chain

Reply:
Ah, ok, ja, a fresh node during the first session it returns `0`